src/adera/Machines/UserControl.cpp

```diff
@@ -36,7 +36,7 @@ const std::string SysMachineUserControl::smc_name = "UserControl";
 
 void MachineUserControl::propagate_output(WireOutput* output)
 {
-    std::cout << "propagate test: " << output->get_name() << "\n";
+    spdlog::info("Propagate test: {}", output->get_name());
```

Comment:
Why are you tracing with a global logger instance here, but you use a specific handle in other places?

Reply:
Because `MachineUSerControl` is not tied to any application, so I believe there is no logger to use, and I don't think it's a good idea to create a whole new logger just for this.